packages/fiori/src/types/MediaGalleryItemLayoutType.js

```diff
@@ -0,0 +1,41 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.fiori.types.MediaGalleryItemLayoutType.prototype
+ * @public
+ */
+const MediaGalleryItemLayoutTypes = {
+
+	/**
+	 * TODO
+	 * @public
+	 * @type {Square}
+	 */
+	 Square: "Square",
+
+	/**
+	 * TODO
+	 * @public
+	 * @type {Wide}
+	 */
+	Wide: "Wide",
```

Comment:
Wide and Square somehow don't match to me, but we can brainstorm about it and if we can't come up with something else, we will keep it.